sputnikdao2/src/policy.rs

```diff
@@ -393,6 +404,47 @@ impl Policy {
     }
 }
 
+/// Walks down the policy in JSON format and updates part of it with new value,
+pub(crate) fn update_policy(policy: &Policy, updates: &[PolicyUpdate]) -> Policy {
+    let mut policy_value = near_sdk::serde_json::to_value(policy).expect("Failed to deserialize");
+    for update in updates {
+        let parts: Vec<&str> = update.path.split("/").collect();
+        let mut item = &mut policy_value;
+        let new_value_item =
+            near_sdk::serde_json::from_str::<near_sdk::serde_json::Value>(&update.new_value)
+                .expect("ERR_POLICY_VALUE");
+        // Walk the path to the item to change.
+        for i in 0..parts.len() {
+            let part = parts[i];
+            match item {
```

Comment:
Does this implementation work for adding a new group to the policy? As stated in https://hackmd.io/lMmmRVXASuGf9Xrx75Pmlw?view#Add-proposal-to-change-policy with the `influencers` group.

If not, I think it doesn't solve the original issue stated in https://github.com/near-daos/sputnik-dao-contract/issues/59